cmd/ebrelayer/types/types.go

```diff
@@ -54,7 +54,7 @@ type EthereumEvent struct {
 	To                    []byte
 	Symbol                string
 	Name                  string
-	Decimals              int32
+	Decimals              int64
```

Comment:
In RegistryEntry, the decimals defined as int64, just change to the same, then avoid type conversion.

message RegistryEntry {
  bool is_whitelisted = 1;
  int64 decimals = 2;

Reply:
@banshee This is something we had to change (and as such will be a merge conflict) in future/devenv-rebased because the unmarshal process was throwing a fit if they where not the exact same types in the integration test. So I will have to look into this when we merge in peggy2 back into devenv-rebased...

Reply:
Switched to int32 here: #1903. we can use int64(n) to cast it before usage and store it as int32